<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(170, 255, 255, 255), stop:0.0113636 rgba(167, 199, 231, 255), stop:1 rgba(0, 128, 128, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 60pt &quot;Corbel&quot;;
text-decoration: underline;
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>RollSmart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signInButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>480</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
font: 30pt &quot;Corbel&quot;;
color: rgb(255,255,255);
background-color: rgb(147, 136, 162)</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Corbel&quot;;
text-decoration: underline;
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Please Sign In</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Corbel&quot;;
text-decoration: underline;
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Corbel&quot;;
text-decoration: underline;
color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailEntry">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,0);
font: 20pt &quot;Corbel&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordEntry">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>410</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);
font: 20pt &quot;Corbel&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="InvalidSignInAttempt">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Corbel&quot;;
color: rgb(255,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="createNewAccountButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>630</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
font: 30pt &quot;Corbel&quot;;
color: rgb(255,255,255);
background-color: rgb(147, 136, 162)</string>
    </property>
    <property name="text">
     <string>Create New Account</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>EmailEntry</tabstop>
  <tabstop>PasswordEntry</tabstop>
  <tabstop>signInButton</tabstop>
  <tabstop>createNewAccountButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
